packages/optimizers/image/package.json

```diff
@@ -39,6 +39,7 @@
     "detect-libc": "^1.0.3"
   },
   "devDependencies": {
+    "@types/node": "^12.12.6",
```

Comment:
Not sure how flow works but this should probably be added to packages that specifically need node types rather than relying on the global package on the top level